app/views/candidate_interface/sign_in/expired.html.erb

```diff
@@ -7,7 +7,7 @@
         <%= t('authentication.expired_token.heading') %>
       </h1>
       <p class="govuk-body">
-        Sign in links only work once and expire after one hour, so you need to request a new one. We’ll send this by email.
+        The sign in link expires after one hour and only works once so you need a new link.
```

Comment:
```suggestion
        The sign in link expires after one hour and only works once, so you need a new link.
```

IMHO the comma reduces ambiguity

Reply:
Thanks! I was too busy worrying about the tests passing...